Network/Messages/Auth/Auth.cpp

```diff
@@ -0,0 +1,68 @@
+/*
+** EPITECH PROJECT, 2024
+** R-Type Network
+** File description:
+** AuthMessage
+*/
+
+#include "Auth/Auth.hpp"
+
+/**
+ * @namespace ABINetwork
+ * @brief Contains functions to manage network communication and operations.
+ */
+namespace ABINetwork
+{
+
+std::vector<asio::detail::buffered_stream_storage::byte_type> & AuthMessage::createLoginPayload(std::string userName, std::string password)
```

Comment:
For me, you should define the type as Paylod to avoid having deadly long types
```suggestion
#define Payload std::vector<asio::detail::buffered_stream_storage::byte_type> &
```

> [!Note]
> Don't forget to put the define in the hpp file ;)